lib/zold/commands/remote.rb

```diff
@@ -89,6 +91,21 @@ def run(args = [])
         o.bool '--reboot',
           'Exit if any node reports version higher than we have',
           default: false
+        # @todo #292:30min Group options by subcommands
+        #  Having all the options in one place _rather than grouping them by subcommands_
+        #  makes the help totally misleading and hard to read.
+        #
+        #  Not all the options are valid for every command - that's the key here.
+        #
+        #  The option below (`--max-nodes`) is an example.
+        #
+        #  **Next actions:**
+        #  - Implement the suggestion above.
+        #  - Remove note from the --max-nodes option saying that it applies to the select
+        #    subcommand only.
+        o.integer '--max-nodes',
+          'This applies only to the select subcommand. Number of nodes to limit to. Defaults to 16.',
```

Comment:
@botanicus what if `Remotes::MAX_NODES` changes tomorrow? This line will not be valid anymore, right? In general, avoid using hard-coded constants anywhere in the code.